.gitignore

```diff
@@ -1 +1,2 @@
 fake-gcs-server
+/.idea
```

Comment:
I recommend having editor-specific ignore entries set via global gitignore files rather than per-project.

Can you remove this?